### DISCUSSION TITLE
horse-upload

### DISCUSSION THREAD

**Discussion by WhebertonFialho** [HashLoad/horse#263] [Q&A] [answered]
Bom dia, gostaria como ficou o upload apos a retirada do THorseHackRequest que é usado no componente [horse-upload](https://github.com/giorgiobazzo/horse-upload) pois esta ocorrendo erro ao tentar compilar com a nova versão..

**Comment by viniciussanchez** [comment, marked as answer]
Dê uma olhada nessa issue aberta no middleware em questão:
https://github.com/giorgiobazzo/horse-upload/issues/6
